rmf_task_sequence/src/rmf_task_sequence/events/utils.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include "utils.hpp"
+
+namespace rmf_task_sequence {
+namespace events {
+namespace utils {
+
+std::string waypoint_name(
```

Comment:
You can simplify this now to just use [`rmf_task::standard_waypoint_name`](https://github.com/open-rmf/rmf_task/blob/c6b9ed4c62d57c1874db572c75b1d079173c6c3e/rmf_task/include/rmf_task/Header.hpp#L65-L67).